src/services/uploader/uploading.controller.test.ts

```diff
@@ -27,6 +29,8 @@ describe('UploadingController', () => {
     mongod = await MongoMemoryServer.create();
     const uri = mongod.getUri();
 
+    process.env.JWT_PRIVATE_KEY = 'ac746c4dc9faf199d7fec029f1e8646c08da3698d9c95b931a1df2ceb666e336dbdacf46763a89777206cf48fc43be42cbe0f988e4bd4a10e7610173d29310ea987d93bae49f6391b91a5338cffbf2389797d7217903b2db1cbf983632f64e088fb515537262d2475589370fc1f5aa7820c34f0f5523fb88f75dace392d22caf'
```

Comment:
Just want to double check that this is different from any production secrets

Reply:
its the same as the other mock one

Reply:
Yup, I just wanted to mark all instances of this